src/figuresClasses.ts

```diff
@@ -1,19 +1,69 @@
 export interface Figure {
-
+  color: 'red' | 'green' | 'blue';
+  shape: string;
```

Comment:
Declare separate enums or unions for this.
```suggestion
  color: Color;
  shape: Shape;
```